src/genie/libs/parser/nxos/show_fex.py

```diff
@@ -0,0 +1,61 @@
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+
+# Python
+import re
+
+# ==============================
+# Schema for 'show fex'
+# ==============================
+
+class ShowFexSchema(MetaParser):
+
+    ''' Schema for "show fex" '''
+
+    schema = {
+        'fexes': {
+            Any(): {
+                'description': str,
+                'state': str,
+                'model': str,
+                'serial_number': str
+            }
+        }
+    }
+
+# ==============================
+# Parser for 'show fex'
+# ==============================
+
+class ShowFex(ShowFexSchema):
+
+    ''' Parser for "show fex"'''
+
+    cli_command = 'show fex'
+
+    def cli(self, output=None):
+        if output is None:
+            # Get output from device
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output 
```

Comment:
```suggestion
        if output is None:
            output = self.device.execute(self.cli_command)
```
then on line 48 you can do:
``` python
for line in output.splitlines():